### PR TITLE
Added pagination for posts

### PR DESCRIPTION
**Pull request by egonm12**

### CODE REVIEW

File: backend/db/seeds.rb [hunk 1]
```diff
@@ -28,9 +28,9 @@
 
 10.times do |i|
```
Comment (StefanPahlplatz) [inline backend/db/seeds.rb]: Only 10 posts?
```suggestion
100.times do |i|
```

File: backend/db/seeds.rb [hunk 2]
```diff
@@ -28,9 +28,9 @@
...
   Post.create(
-      sender: User.second,
-      receivers: User.last(3),
-      message: Faker::Company.bs,
-      kudos: rand(1..500)
+    sender: User.second,
+    receivers: User.last(3),
```
Comment (StefanPahlplatz) [inline backend/db/seeds.rb]: Always the last 3 users?

File: backend/spec/graphql/connections/posts_connection_spec.rb [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
   let!(:users) { create_list(:user, 3) }
   let!(:posts) { create_list(:post, 3, sender: users.first, receivers: [users.second, users.last]) }
-  it 'has a :totalCount field that returns a Int type' do
+  it "has a :totalCount field that returns a Int type" do
     expect(subject).to have_field(:totalCount).that_returns(types.Int)
   end
-  xit 'returns the number of nodes' do
-    #TODO write this test
+  xit "returns the number of nodes" do
+    # TODO write this test
```
Comment (StefanPahlplatz) [inline backend/spec/graphql/connections/posts_connection_spec.rb]: TODO

File: backend/db/seeds.rb (hunk 2 above)
Comment (egonm12) [reply backend/db/seeds.rb]: Make it random? 

```suggestion
    receivers: User.limit(rand(1..5)).order("RANDOM()"),
```

Comment (StefanPahlplatz) [reply backend/db/seeds.rb]: I think that would be better, this would also include the sender though.

Comment (egonm12) [reply backend/db/seeds.rb]: Maybe a bit hacky, suggestions?

```suggestion
    receivers: User.limit(rand(1..4)).order("RANDOM()").where.not(id: 1),
```

Comment (StefanPahlplatz) [reply backend/db/seeds.rb]: Looks good to me, you could consider making the values named constants, so you don't have magic numbers in the tests.

**Review by StefanPahlplatz — APPROVED** [review]

**Pull request merged**